### DISCUSSION TITLE
网上一堆冒牌货

### DISCUSSION THREAD

**Discussion by ProgramCX** [win12-online/win12#492] [General]
![屏幕截图 2024-10-26 203205](https://github.com/user-attachments/assets/cac6ff50-2d67-4b4c-b190-ec5a6849b74d)
把我整笑了

**Comment by ProgramCX** [comment]
https://win12online.com/

**Comment by lingbopro** [comment]
不太对劲，这玩意属于是偷了素材，但是代码用React写的

**Reply by lingbopro** [reply]
这玩意非常粗制滥造，应用全是套iframe，甚至最小化都没实现

**Reply by kjmjh** [reply]
其实你在必应上一搜:win12网页版，出来很多冒牌的，甚至有人还接入广告…

**Reply by ProgramCX** [reply]
是的

**Reply by ProgramCX** [reply]
> 其实你在必应上一搜:win12网页版，出来很多冒牌的，甚至有人还接入广告…

遇到过

**Reply by Mbilse** [reply]
> > 其实你在必应上一搜:win12网页版，出来很多冒牌的，甚至有人还接入广告…
> 
> 遇到过
> 

我也

**Comment by zsr-lukezhang** [comment]
![image](https://github.com/user-attachments/assets/b57555de-86f0-4684-94f7-f8656cee6ce0)
这个是最大的冒牌货
https://win12.gitapp.cn/
![image](https://github.com/user-attachments/assets/cba82e55-697b-4314-9234-50e286313d04)
一大堆广告

**Reply by lingbopro** [reply]
难绷

**Reply by zsr-lukezhang** [reply]
> 难绷

帮助创作者创收 ❌
帮助搬运者创收 ✔

**Reply by lingbopro** [reply]
> > 难绷
> 
> 帮助创作者创收 ❌
> 帮助搬运者创收 ✔

创收 ✖️
创人 ✔️

**Reply by Mbilse** [reply]
> ![image](https://github.com/user-attachments/assets/b57555de-86f0-4684-94f7-f8656cee6ce0)
> 这个是最大的冒牌货
> https://win12.gitapp.cn/
> ![image](https://github.com/user-attachments/assets/cba82e55-697b-4314-9234-50e286313d04)
> 一大堆广告
来自github帐号 g +好几个e +k
用户

**Comment by zsr-lukezhang** [comment]
![image](https://github.com/user-attachments/assets/20135958-ae12-42fd-8404-637ceb81e456)
甚至还有 Copyright

**Reply by lingbopro** [reply]
emmm...那个难绷的win12是在这玩意的子域下的是吧

**Reply by zsr-lukezhang** [reply]
> emmm...那个难绷的win12是在这玩意的子域下的是吧

是的

**Reply by lingbopro** [reply]
我感觉这两个东西好像没啥直接关系

**Reply by zsr-lukezhang** [reply]
> 我感觉这两个东西好像没啥直接关系

什么意思（没看懂）

**Reply by zsr-lukezhang** [reply]
![image](https://github.com/user-attachments/assets/509ba63e-4a26-4d35-827e-bc38eaf83915)
看第三行

**Reply by lingbopro** [reply]
> ![image](https://github.com/user-attachments/assets/509ba63e-4a26-4d35-827e-bc38eaf83915)
> 看第三行

艹

**Reply by zsr-lukezhang** [reply]
> 艹


我都不敢想象剩下的工具都是哪里来的

**Reply by lingbopro** [reply]
> > 艹
> 
> 
> 我都不敢想象剩下的工具都是哪里来的

别的倒还好，大部分都是纯工具性质的

**Reply by zsr-lukezhang** [reply]
> > > 艹
> > 
> > 
> > 我都不敢想象剩下的工具都是哪里来的
> 
> 别的倒还好，大部分都是纯工具性质的

我的意思是，这些都是从哪里抄的

**Reply by zsr-lukezhang** [reply]
![image](https://github.com/user-attachments/assets/1249e767-4e71-494c-8132-5dded8460a54)
《金字塔》

**Reply by lingbopro** [reply]
> ![image](https://github.com/user-attachments/assets/1249e767-4e71-494c-8132-5dded8460a54)
> 《金字塔》

引用回复多了就会这样（）

**Reply by lingbopro** [reply]
> > > > 艹
> > > 
> > > 
> > > 我都不敢想象剩下的工具都是哪里来的
> > 
> > 别的倒还好，大部分都是纯工具性质的
> 
> 我的意思是，这些都是从哪里抄的

这些工具性质的东西估计大多是开源的，而且也没多复杂，抄几个到还好

**Reply by Mbilse** [reply]
我个艹！！！

**Reply by kj23qaq** [reply]
我把自己开源了(bs

**Reply by Mbilse** [reply]
> ![image](https://github.com/user-attachments/assets/1249e767-4e71-494c-8132-5dded8460a54)
> 《金字塔》

我还有更酷的
>>>>>>>>>>>>>>>>>>>>>>>>>>>> .
>>>>>>>>>>>>>>>>>>>>>>>>>>>> .

和

>>>>>>>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>>.

>>>>>>>>>>>>>>.

>>>>>>>>>>>>>.

>>>>>>>>>>>>.

>>>>>>>>>>>.

>>>>>>>>>>.

>>>>>>>>>.

>>>>>>>>.

>>>>>>>.

>>>>>>.

>>>>>.

>>>>.

>>>.

>>.

>.

.

**Reply by kjmjh** [reply]
hh

**Reply by freedom-323** [reply]
~咳咳 这不左右互搏吗~

**Comment by zsr-lukezhang** [comment]
![image](https://github.com/user-attachments/assets/d9312579-0f64-4f93-ba98-2a3e15fd6d8a)
《Edge 默认用 Google》

**Reply by ProgramCX** [reply]
绝了

**Reply by lingbopro** [reply]
神金

**Reply by kjmjh** [reply]
无敌了

**Reply by Mbilse** [reply]
> ![image](https://github.com/user-attachments/assets/d9312579-0f64-4f93-ba98-2a3e15fd6d8a)
> 《Edge 默认用 Google》

艹,绝

**Comment by starry-source** [comment]
物质不齐，物之情也。

**Reply by zsr-lukezhang** [reply]
> 物质不齐，物之情也。

？没看懂（

**Reply by starry-source** [reply]
> ？没看懂（

自己搜。

**Reply by zsr-lukezhang** [reply]
> > ？没看懂（
> 
> 自己搜。

![image](https://github.com/user-attachments/assets/2e5c0d20-286c-4a37-ac8d-2bdb53981bfe)

![image](https://github.com/user-attachments/assets/0e4b93d1-19bf-41fc-a17d-ac33194ebd01)

**Reply by Mbilse** [reply]
> > > ？没看懂（
> > 
> > 自己搜。
> 
> ![image](https://github.com/user-attachments/assets/2e5c0d20-286c-4a37-ac8d-2bdb53981bfe)
> 
> ![image](https://github.com/user-attachments/assets/0e4b93d1-19bf-41fc-a17d-ac33194ebd01)
> 
> 
> 

wow

**Comment by Mbilse** [comment]
a...
________________________________

**Comment by CRH380AL-2542** [comment]
这类冒牌货还算好的，起码正经地写了一个网页端操作系统。项目火了之后，看看360上都是些什么乱七八糟的玩意
![image](https://github.com/user-attachments/assets/c7e7f847-c111-4e0b-bd67-cb9da2710ab8)
这玩意更好笑

**Reply by tjy-gitnub** [reply]
能安吗（好奇 -u-)o

**Reply by lingbopro** [reply]
> 能安吗（好奇 -u-)o

大概率是广告+p2p下崽器

**Reply by kjmjh** [reply]
《w.dqwer88.top 广告 2025-5》

**Reply by CRH380AL-2542** [reply]
不是，就是各种精简版系统+Ghost，和Windows12半毛钱关系没有

**Reply by Mbilse** [reply]
> 不是，就是各种精简版系统+Ghost，和Windows12半毛钱关系没有

哦，原来还有人用360

**Comment by Mbilse** [comment]
这里离题一下：
是否允许我在Mbilse Startpage 搜索引擎  https://mbilsesch.github.io   上展示有关Win12网页版地址搜索链接？

**Reply by tangyuan0821** [reply]
可以

**Comment by tangyuan0821** [comment]
现在可以举报了，因为它偷的素材是使用CC-BY-SA 4.0许可的。CC-BY-SA 4.0协议规定，使用者必须给出适当的署名，提供指向本许可协议的链接，注明是否进行了修改 。

**Reply by tangyuan0821** [reply]
它似乎一项也没有满足。

**Reply by lingbopro** [reply]
但是它偷的是采用CC之前的版本吧，那CC是否依然生效？

**Reply by tangyuan0821** [reply]
> 但是它偷的是采用CC之前的版本吧，那CC是否依然生效？

依照著作权法，即使没有使用任何许可证。只要对方未经许可使用图片的行为就可能构成侵权。

**Reply by lingbopro** [reply]
但是之前版本整体采用EPL，可否看作EPL也对美术内容生效？
（虽然它也没有遵循EPL就是了

**Reply by tangyuan0821** [reply]
> 但是之前版本整体采用EPL，可否看作EPL也对美术内容生效？
> （虽然它也没有遵循EPL就是了

生效，需按照EPL规定来使用和分发。 玄武湖畔 ***@***.***

**Comment by freedom-323** [comment]


**Reply by tangyuan0821** [reply]
这个镜像的版本没有违反开源协议，他有署名。
https://github.com/lttthedev/lttthedev.github.io

**Reply by tangyuan0821** [reply]
不过他的翻译很粗糙，像机翻而且没有校对过

**Comment by freedom-323** [comment]
https://www.heyistore.com/

**Reply by tangyuan0821** [reply]
这种早期版本就不管了，对于我们也没有多大的影响

**Comment by Mbilse** [comment]
这是yandex搜索:

![Screenshot_2026-04-28-20-54-55-71.jpg](https://github.com/user-attachments/assets/5e1ad7af-cdfa-4b38-a07f-91f5794e04ef)
